rest/specs/src/main/filtered-resources/openapi/openapi.yaml

```diff
@@ -186,6 +317,48 @@ components:
       maxLength: 80
       pattern: '^([1-9]([0-9]+)?)\.(0|([1-9]([0-9]+)?))(\.(0|([1-9]([0-9]+)?))(-.*)?)?$'
       example: 1.2, 2.4.3, 0.2.5-SNAPSHOT
+    MetadataInformationResponse:
+      type: object
+      required:
+        - transformationStatus
+        - metadataInformations
+      properties:
+        transformationStatus:
+          $ref: '#/components/schemas/Status'
+        metadataInformations:
```

Comment:
❗️ Might want to also add more details about the status in case of failure. Similar to what we do for the default error message. Same thing for `MetadataInformation`